src/gcp_scanner/arguments.py

```diff
@@ -127,9 +127,13 @@ def arg_parser():
 
   args: argparse.Namespace = parser.parse_args()
 
-  if not args.key_path and not args.gcloud_profile_path \
-    and not args.use_metadata and not args.access_token_files\
-    and not args.refresh_token_files:
+  if not [arg for arg in
```

Comment:
Pretty good but can be even clearer with:
`if not any(bool(arg) for arg in [...]):`